src/components/visualizations/chart/highcharts/plugins/test/getTickPositions.spec.ts

```diff
@@ -608,4 +609,19 @@ describe('get tick positions for dual axes chart', () => {
             });
         });
     });
+
+    describe('createArrayFromRangeWithMiddleZero', () => {
+
+        const runTest = (start: number, end: number, ticksNumber: number = TICK_AMOUNT) => {
+            const result = createArrayFromRangeWithMiddleZero(start, end, ticksNumber);
+
+            expect(result.indexOf(0)).not.toEqual(-1);
+            expect(result.length).toEqual(ticksNumber);
+        };
+
+        it('should add zero between start and end values', () => {
```

Comment:
We had no better way how to do this tests before, but now we know 
`it.each` construction for data driven tests, so we should use this and in future rewrite all tests in this way. For details see
https://confluence.intgdc.com/display/~dan.homola/Testing+best+practices+and+guidelines

Reply:
Good to know that :)
I upgraded jest version to support `it.each` function.